STAT/TStatToolkit.cxx

```diff
@@ -1343,7 +1345,8 @@ TMultiGraph * TStatToolkit::MakeMultGraph(TTree * tree, const char *groupName, c
     Double_t meanT,rmsT=0;
     if (gr==NULL){
       ::Error("MakeMultGraph","Not valid sub-expression %s",exprVarArray->At(igraph)->GetName());
-      continue;
+      return 0;  
+//      continue;
```

Comment:
Behavior should be parameter - new parameter to function - error handling in case part of input missing?
Doxygen documentation for function to describe behavior

Reply:
delete allocated resources before return